CONTRIBUTING.md

```diff
@@ -217,5 +217,42 @@ These instructions will help you to set up and use local website authoring and t
 - Open the `http://localhost:4000/` page in your browser.
 - Website will automatically rebuild and refresh in the browser to accommodate the related code changes.
 
+### End-to-end demo helper
```

Comment:
@ytsarev suggestion to promote the header level, as "End-to-end demo helper" is not the child of  "k8gb.io website"
```suggestion
## End-to-end demo helper
```